examples/jsm/nodes/ShaderNode.js

```diff
@@ -376,6 +385,21 @@ export const metalness = new ShaderNodeObject( new PropertyNode( 'Metalness', 'f
 export const alphaTest = new ShaderNodeObject( new PropertyNode( 'AlphaTest', 'float' ) );
 export const specularColor = new ShaderNodeObject( new PropertyNode( 'SpecularColor', 'color' ) );
 
+export const materialAlphaTest = new ShaderNodeObject( new MaterialNode( MaterialNode.ALPHA_TEST ) );
+export const materialColor = new ShaderNodeObject( new MaterialNode( MaterialNode.COLOR ) );
+export const materialOpacity = new ShaderNodeObject( new MaterialNode( MaterialNode.OPACITY ) );
+export const materialSpecular = new ShaderNodeObject( new MaterialNode( MaterialNode.SPECULAR ) );
+export const materialRoughness = new ShaderNodeObject( new MaterialNode( MaterialNode.ROUGHNESS ) );
+export const materialMetalness = new ShaderNodeObject( new MaterialNode( MaterialNode.METALNESS ) );
+
+export const skinning = new ShaderNodeProxy( SkinningNode );
+
+export const lightContext = new ShaderNodeProxy( LightContextNode );
+
+export const toColorSpace = ( node, encoding ) => nodeObject( new ColorSpaceNode( null, nodeObject( node ) ).fromEncoding( encoding ) );
```

Comment:
I think we can let's just `colorSpace()` because "`to`" can be implicit like used in other functions.

Reply:
No, I think `toColorSpace` is better - it explicitly converts a color to a color space.

Reply:
> No, I think toColorSpace is better - it explicitly converts a color to a color space.

That said, you would be assuming that any conversions function have the prefix "to", but we don't have that in either `ShaderNode` or `GLSL`. Use prefix 'to' would make more sense in the context of class function like `color.toColorSpace( ... )`

You can try `linearTosRGB()` too for example

Reply:
I think that if e.g. there would be a function that converts a value to one of the types (like ConvertType or ConstNode) and if there would be a function that converts a value to float, then I would expect them to be called `toType( value, type )` and `float( value )`... And this is analoguous - e.g. `toColorSpace()`, but `linear()` and `srgb()`.

Reply:
I think that this should be `colorSpace( node, source, target )`. 
Unless we detect this source of `ColorSpace` at the compilation level, it makes no much sense to have only the `target` argument.

Reply:
I think this can be added in another PR... OK, I will rename `toColorSpace` to `colorSpace`.